changesets/scripts/generate_changeset.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
suggestion: is there any reason this shouldn't use `env`?

```suggestion
#!/usr/bin/env bash
```